qcodes/instrument_drivers/yokogawa/GS200.py

```diff
@@ -316,6 +376,47 @@ def __init__(self, name: str, address: str, terminator: str = "\n",
         # Reset function
         self.add_function('reset', call_cmd='*RST')
 
+        self.add_parameter(
+            "setpoints_start",
+            initial_value=0,
+            unit="",
+            label="start value for the setpoints",
+            get_cmd=None,
+            set_cmd=None
+        )
+
+        self.add_parameter(
+            "setpoints_stop",
+            initial_value=0,
+            unit="",
+            label="stop value for the setpoints",
+            get_cmd=None,
+            set_cmd=None
+        )
```

Comment:
It does not seem like these setpoints parameters are currently used ?

Reply:
You are right, they are not currently used, so I decided to remove them from this PR.